tests/functional/adapter/tidb/basic/test_tidb.py

```diff
@@ -83,5 +82,5 @@ def expected_catalog(self, project):
         )
 
 
-class TestIncrementalUniqueKey(BaseIncrementalUniqueKey):
+class TestIncrementalNotSchemaChange(BaseIncrementalNotSchemaChange):
```

Comment:
why delete the original test

Reply:
`BaseIncrementalUniqueKey` test is moved to the `incremental` directory.